content/docs/2.10/scalers/selenium-grid-scaler.md

```diff
@@ -23,6 +23,7 @@ triggers:
       browserVersion: '91.0' # Optional. Only required when supporting multiple versions of browser in your Selenium Grid.
       unsafeSsl : 'true' # Optional
       activationThreshold: 5 # Optional
+      platformName: 'Linux' # Optional
```

Comment:
Is it `Linux` or `linux`? In the code there's `linux` used.

Reply:
@zroubalik, the platformName is validaded in a case insensitive way.